.github/ISSUE_TEMPLATE/icon_removal.yml

```diff
@@ -0,0 +1,30 @@
+name: Icon removal
+description: Report an icon for removal
+title: "Remove: "
+labels: [removal request]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `removal request` label][removal-request-issues]
+        (including closed issues) before you continue. If you find one for the brand
+        you're reporting then leave a comment on it or add a reaction.
+
+
+        [removal-request-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22removal+request%22+is%3Aopen+
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Reason for Removal
```

Comment:
I think we should add a description here that asks the reporter to include a link as well if there's one, which should usually be the case.

E.g. we would probably want the reports to look something like "look, the website says the company was closed in XXXX" or "this blogpost announced the brand has been discontinued"; rather than "the website is offline" or "there was a blog post".